BraintreeCore/src/main/java/com/braintreepayments/api/core/UserCanceledException.kt

```diff
@@ -14,7 +14,7 @@ import androidx.annotation.RestrictTo
  * could mean the user returned to the app through multi-tasking without completing the flow,
  * the user closed the browser tab, or the user pressed the back button.
  */
-open class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
+class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
     message: String?,
-    val isExplicitCancelation: Boolean = false
+    val isExplicitCancellation: Boolean = false
```

Comment:
The single `l` is intentional so we will want to revert this

Reply:
Similar thread here: https://github.com/braintree/braintree_android/pull/1058#discussion_r1671204729

Reply:
Ah, sorry Steven and I apparently posted the same thing at the same time!